src/components/FamilyFriendly.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import DestinationList from './DestinationList';
+import BackButton from './BackButton';
+
+import familyFriendlyImage from '../../images/familyFriendlyImage.jpg'; // Import your background image
```

Comment:
path of ../../images/familyFriendlyImage.jpg should be ../../images/FamilyFriendlyImage.jpg